servers/src/main/java/tachyon/master/TachyonMaster.java

```diff
@@ -121,12 +121,8 @@ public TachyonMaster(TachyonConf tachyonConf) {
 
     mIsStarted = false;
     mWebPort = webPort;
-    mMinWorkerThreads =
-        mTachyonConf.getInt(Constants.MASTER_MIN_WORKER_THREADS, Runtime.getRuntime()
-            .availableProcessors());
```

Comment:
Yes this is Intentional. And this is about the minimum number of threads to start with setting it lower will not affect the correctness.